packages/cozy-app-publish/package.json

```diff
@@ -38,5 +38,8 @@
     "setupFiles": [
       "./test/jestLibs/setupJest.js"
     ]
+  },
+  "engines": {
+    "node": ">=6"
```

Comment:
This will stop running the script or just warn and continue?

Reply:
Warn and continue. I think we should not add unnecessary checks in our code. This is the built-in way to do node version checks.

Reply:
Ok, it was just for my culture, let's do it like this 👍